source/unified-test-format/unified-test-format.rst

```diff
@@ -1407,13 +1407,21 @@ The structure of each object is as follows:
 
 - ``hasFailure``: Optional boolean. When this field is set to ``true``, 
   the test runner MUST assert that the log message has a ``failure`` value
-  in its attached data, and that the failure has not been redacted in any
-  way. When this field is set to ``false``, the test runner MUST assert
-  that the log message either has no ``failure`` value at all or that it
-  has a redacted failure, e.g. an empty string or object. 
-  The exact form of those assertions will vary based on the driver's chosen
-  representations for failures and redacted failures in log messages.
+  in its attached data. Then this field is set to ``false``, the test runner
```

Comment:
Then -> When

Reply:
thanks; ended up deleting this per above thread about using a single boolean